examples/notebooks/bots_on_solidity.ipynb

```diff
@@ -359,19 +380,35 @@
     "fixed_math_address = sol_agents[\"agent_0\"].deploy(project.MockFixedPointMath)\n",
     "fixed_math = project.MockFixedPointMath.at(fixed_math_address)\n",
     "\n",
-    "base_ERC20.mint(int(config.target_liquidity * 10**18), sender=sol_agents[\"agent_0\"])\n",
+    "hyperdrive_math_address = sol_agents[\"agent_0\"].deploy(project.MockHyperdriveMath)\n",
+    "hyperdrive_math = project.MockHyperdriveMath.at(hyperdrive_math_address)\n",
+    "\n",
+    "base_ERC20.mint(int(config.target_liquidity * 2 * 10**18), sender=sol_agents[\"agent_0\"])\n",
```

Comment:
I thought this helped avoid a crash, but I may have been hallucinating. will need to test more thoroughly. it makes more sense that running out of Eth instead of ERC20 would potentially cause a crash.